code/modules/mining/mine_items.dm

```diff
@@ -263,6 +263,268 @@
+			log_admin("[key_name(usr)] activated a bluespace capsule away from the mining level at [T.x], [T.y], [T.z]")
+		load()
+		new /obj/effect/effect/smoke(loc)
+		qdel(src)
+
+/obj/item/weapon/survivalcapsule/proc/load()
+	var/list/blacklist = list(/area/shuttle) //Shuttles move based on area, and we'd like not to break them
+	var/turf/start_turf = get_turf(loc)
+	var/turf/cur_turf
+	var/x_size = 5
+	var/y_size = 5
+	var/list/walltypes = list(/turf/simulated/wall/survival)
+	var/floor_type = /turf/simulated/floor/pod
+	var/room
+
+	//Center the room/spawn it
+	start_turf = locate(start_turf.x -2, start_turf.y - 2, start_turf.z)
+
+	room = spawn_room(start_turf, x_size, y_size, walltypes, floor_type, "Emergency Shelter")
+
+	start_turf = get_turf(loc)
+
+	//Fill it
+
+	//The door
+	cur_turf = locate(start_turf.x, start_turf.y-2, start_turf.z)
+	new /obj/machinery/door/airlock/survival_pod(cur_turf)
+
+
+	//Bed middle right
+	cur_turf = locate(start_turf.x+1, start_turf.y, start_turf.z)
+	new /obj/structure/stool/bed/pod(cur_turf)
+	new /obj/item/weapon/bedsheet/brown(cur_turf)
+
+	//Chair bottom right
+	cur_turf = locate(start_turf.x+1, start_turf.y-1, start_turf.z)
+	new /obj/structure/tubes(cur_turf)
+	var/obj/structure/stool/bed/chair/comfy/black/C = new (cur_turf)
+	C.dir = 8
+
+	//GPS computer top right
+	cur_turf = locate(start_turf.x+1, start_turf.y+1, start_turf.z)
+	new /obj/item/device/gps/computer(cur_turf)
+
+	//Donk Pocket Storage Top/middle
+	cur_turf = locate(start_turf.x, start_turf.y+1, start_turf.z)
+	new /obj/machinery/smartfridge/survival_pod(cur_turf)
+
+	//Table in Bottom Left
+	cur_turf = locate(start_turf.x-1, start_turf.y-1, start_turf.z)
+	new /obj/structure/table/survival_pod(cur_turf)
+
+	//Sleeper Middle Left
+	cur_turf = locate(start_turf.x-1, start_turf.y, start_turf.z)
+	new /obj/machinery/sleeper/survival_pod(cur_turf)
+
+	//Fans Top Left
+	cur_turf = locate(start_turf.x-1, start_turf.y+1, start_turf.z)
+	new /obj/structure/fans(cur_turf)
+
+	//Signs
+	cur_turf = locate(start_turf.x-2, start_turf.y, start_turf.z)
+	var/obj/structure/sign/mining/survival/S1 = new(cur_turf)
+	S1.dir = EAST
+
+	cur_turf = locate(start_turf.x+2, start_turf.y, start_turf.z)
+	var/obj/structure/sign/mining/survival/S2 = new(cur_turf)
+	S2.dir = WEST
+
+	cur_turf = locate(start_turf.x, start_turf.y+2, start_turf.z)
+	var/obj/structure/sign/mining/survival/S3 = new(cur_turf)
+	S3.dir = NORTH
+
+	cur_turf = locate(start_turf.x-1, start_turf.y-2, start_turf.z)
+	var/obj/structure/sign/mining/survival/S4 = new(cur_turf)
+	S4.dir = SOUTH
+
+	cur_turf = locate(start_turf.x+1, start_turf.y-2, start_turf.z)
+	new /obj/structure/sign/mining(cur_turf)
```

Comment:
There's nothing in the runtime viewer, nothing printed to world, and nothing in DD printed that was an error, warning, or anything of any sort. It literally just fails.

Placing the template with the verb works fine (and uses the same, exact verb).

Another problem the template loader has is that it also cannot properly determine safe vs unsafe turfs; it pretty much would register any further as being safe.